components/Image/image.html

```diff
@@ -1,20 +1,66 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/829cb68a23.js" crossorigin="anonymous"></script>
+    <link rel="icon" type="image/png" href="../images/favicon-32x32.png" />
     <link rel="stylesheet" href="./image.css">
     <title>Image</title>
 </head>
+
 <body>
-    <h2>Responsive Image</h2>
-    <div class="img-responsive-container">
-        <img src="../images/image-res.jpg" alt="image-responsive" class="img-responsive">
-    </div>
-    <h2>Cirle Image</h2>
-    <div class="img-circle-container">
-        <img src="../images/avatar-1.jpg" alt="image-round" class="img-responsive img-circle">
-    </div>
+    <main>
+
+        <nav class="header">
+            <a href="../../index.html" class="title">
+                <h2>CheckMate UI</h2>
+            </a>
+            <button class="menu-bar"><i class="fas fa-bars"></i></button>
+        </nav>
+
+        <section>
+            <h2 class="page-title">Image</h2>
+            <p class="section-para">An image is graphical representation of something to make a website much more
+                appealing. Images are generally responsive which expands with the width of the container they are in.
+            </p>
+        </section>
+
+        <section>
+            <h2 class="section-heading">Responsive Image</h2>
+            <p class="section-para">Make any image responsive by adding <span class="colored-text">img-responsive</span>
+                class to it and fit
+                it inside container of any width and it will expand by its container's width.</p>
```

Comment:
Indentation could be improved.
e.g.

```HTML
<p class="section-para">
    Make any image responsive by adding
    <span class="colored-text">img-responsive</span>
    class to it and fit it inside container of any width and it will expand by its container's width.
</p>
```

Reply:
done